ztp/ran-site-plan-crd/site-config-crd.yaml

```diff
@@ -0,0 +1,298 @@
+                          ServiceNetwork is the list of IP address pools for services default is 172.30.0.0/16.
+                          currently only one entry is supported.
+                        type: array
+                        maxItems: 1
+                        items:
+                          type: string
+                      apiVIP:
+                        description: APIVIP is the virtual IP used to reach the OpenShift cluster's API.
+                        type: string
+                      ingressVIP:
+                        description: IngressVIP is the virtual IP used for cluster ingress traffic.
+                        type: string
+                      manifestsConfig:
+                        description: day0 required manifests Config.
+                        type: object
+                        properties:
+                          ntpServer:
+                            description: ntpServer required by chrony manifest.
+                            type: string
+                      additionalNTPSources:
+                        description: |
+                          AdditionalNTPSources is a list of NTP sources (hostname or
+                          IP) to be added to all cluster hosts. They are added to any
+                          NTP sources that were configured through other means.
+                        type: array
+                        items:
+                          type: string
+                      ignitionConfigOverride:
+                        description: Json formatted string containing the user overrides for the initial ignition config.
+                        type: string
+                      nodes:
+                        description: List of the nodes that belong to the cluster.
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            hostName:
+                              description: hostName that will get applied to the node.
+                              type: string
+                            bmcAddress:
+                              description: |
+                                Address holds the URL for accessing the controller on the network. Must contain the redfish or the HW identification in the
+                                url for example redfish-virtualmedia+https://12.13.222.2/redfish/v1/Systems/System.Embedded.1
+                                example for Dell idrac-virtualmedia+https://10.16.231.87/redfish/v1/Systems/System.Embedded.1
+                              type: string
+                            bmcCredentialsName:
+                              description: Bmc credential secret info
+                              type: object
+                              properties:
+                                name:
+                                  description: "Name of the bmcCredentials secret"
```

Comment:
Should this specify the format of the bmcCredential secret contents?  I'm assuming it should contain the username and the password, but how should these be encoded in the secret?